src/base/opt/cookie.rs

```diff
@@ -359,6 +359,14 @@ impl hash::Hash for ClientCookie {
     }
 }
 
+//--- PartialEq
+
+impl PartialEq for ClientCookie {
```

Comment:
Why does PartialEq need to be implemented manually?

Reply:
No idea, but that wasn't added/changed in this PR.

Reply:
It comes from commit b9e581910e3d088fecdd05a2d5b3dc3ca23e06d3 which is part of this PR.

Reply:
Oh. My apologies. Then it will be replaced with a Derive or removed entirely if not needed, I don't recall that change at all.